recipe/meta.yaml

```diff
@@ -1,50 +1,104 @@
+  # supporting py3.8+
+  skip: True  # [py<38]
+  # no rust nightly compiler for s390x
+  skip: true  # [s390x]
+  # skipping windows because missing maturin >=1.3.2 and since win is not a priority for SF
+  skip: true  # [win]
 
 requirements:
   build:
-    - python                              # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-    - crossenv                            # [build_platform != target_platform]
-    - maturin >=0.14,<0.15                # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
-    - posix                               # [win]
+    - python
+    - posix   # [win]
     - cmake
-    - make                                # [unix]
+    - make    # [unix]
     - cargo-bundle-licenses
   host:
     - python
     - pip
-    - maturin >=0.14,<0.15
+    - maturin 1.5.1
   run:
     - python
     - numpy >=1.16.0
-    - backports.zoneinfo                   # [py<39]
-    - typing_extensions >=4.0.0            # [py<311]
-    - packaging                            # [py>=310]
+
+# skipping test_delta because of missing fastexcel and ezodf
+{% set tests_modules_to_ignore = " --ignore=py-polars/tests/unit/io/test_spreadsheet.py" %}
+# skipping test_delta because of missing adbc_driver_sqlite
+{% set tests_modules_to_ignore = tests_modules_to_ignore + " --ignore=py-polars/tests/unit/io/database/test_read.py --ignore=py-polars/tests/unit/io/database/test_write.py" %}
+# skipping test_delta because of some authentication problem (403 forbidden)
+{% set tests_modules_to_ignore = tests_modules_to_ignore + " --ignore=py-polars/tests/unit/io/cloud/test_aws.py" %}
+
+# skipping problematic tests without clear troubleshooting
+{% set tests_to_skip = "test_namespace_warning_on_override or test_cse_expr_selection_context or test_extension" %}
+{% set tests_to_skip = tests_to_skip + " or test_from_dataframe_pandas_parametric or test_from_dataframe_pandas_zero_copy_parametric" %}
+{% set tests_to_skip = tests_to_skip + " or test_from_dataframe_pandas_timestamp_ns or test_streaming_sort" %}
+
+# skipping tests that fail on py3.8 and osx most likely because of incorrect handling of underlying types in pandas.DataFrames
+# (not related to polars)
+{% set tests_to_skip = tests_to_skip + " or test_from_dataframe_pandas_native_parametric or test_from_dataframe_pandas_native_zero_copy_parametric" %}  # [osx and py==38]
+{% set tests_to_skip = tests_to_skip + " or test_to_dataframe_pandas_boolean_subchunks or test_from_pandas_index or test_to_pandas" %}                  # [osx and py==38]
+
+# skipping tests that fail on prefect because of missing avro files
+{% set tests_to_skip = tests_to_skip + " or test_scan_iceberg_plain or test_scan_iceberg_filter_on_partition or test_scan_iceberg_filter_on_column" %}
+
+# skipping this import test, because it is an upstream issue as documented in 
```

Comment:
missing the link https://github.com/pola-rs/polars/issues/14442

Reply:
I already committed this locally and will push as soon as all tests pass (build+test takes way too long at every push).